src/controllers/users.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const { UserModel } = require('../models/users');
+
+const getAllUsers = (req, res) => {
+  const users = UserModel.getUsers();
+
+  res.send(users);
+};
+
+const addNewUser = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.status(400).send('Bad request');
+
+    return;
+  }
+
+  const newUser = UserModel.addUser({ name });
+
+  res.statusCode = 201;
+  res.send(newUser);
```

Comment:
```suggestion
  res.status(201).send(newUser);
```

Consider maintaining consistency in your code (if you are using the chaining style for error response, so continue using it)